single-work.php

```diff
@@ -0,0 +1,74 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+
+<?php
+if (have_posts()) {
+  while (have_posts()) {
+    the_post();
+
+    $work_cats = wp_get_post_terms($post->ID, 'work_cat');
+    $work_artists = get_post_artists($post->ID);
```

Comment:
is this one of our utility functions? Perhaps we should start to prefix them `igv_` as it looks a bit like a core wp function so could send someone the wrong way to debug